frontend/src/react/Copy/ConfirmCopy/ConfirmCopy.js

```diff
@@ -0,0 +1,90 @@
+import React, {Component} from 'react';
+import connect from 'react-redux/es/connect/connect';
+import moment from 'moment';
+
+import ConfirmCopyDisplayer from './ConfirmCopyDisplayer';
+import {patchCopy} from '../../../repository/copies';
+import {Redirect} from 'react-router';
+
+const COPIES_KEY = 'irate_copies';
```

Comment:
Ideally it would be better to have a unique key for each student, because if multiple students use the same browser they would share the same local storage

Reply:
If they use the same browser they can't answer at the same time, nop ?

Reply:
True, but let's say : Student 1 begins his exam, leaves the computer
Student 2 logs in, using the same browser, his localstorage will load with student 1's copy

Reply:
Yeah, but that's 1/1000000000000 scenario no ? Let's keep it like this for now

Reply:
Sure, I juste talk about that cause I had to do it myself on another project :D

Reply:
I don't think so @NansD . I'd rather have a nested object inside the irate_copies local storage that identifies the userID. Then JSON.stringify the whole object.